include/language.php

```diff
@@ -345,6 +345,22 @@ public function __get( $property ) {
 			return $this->term_props[ $term_prop_type ][ $term_prop ];
 		}
 
+		if ( 'search_url' === $property || 'home_url' === $property ) {
+			/** This filter is documented in wordpress/wp-includes/functions.php */
+			if ( WP_DEBUG && apply_filters( 'deprecated_function_trigger_error', true ) ) {
+				trigger_error( // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_trigger_error
+					sprintf(
+						"Class property %1\$s::\$%2\$s is deprecated, use %1\$s::get_%2\$s() instead.\nError handler",
+						esc_html( get_class( $this ) ),
+						esc_html( $property ),
+					),
+					E_USER_DEPRECATED
+				);
+			}
+
+			return $this->{"get_{$property}"}();
+		}
```

Comment:
Could we avoid some duplicate code in the method?

Reply:
Indeed, done in f5db54d.